generator/generator.py

```diff
@@ -284,7 +284,8 @@ def generate_boards_json(input_data, arduino_cli_path):
                 boards[fqbn]["name"] = board["name"]
                 break
 
-        boards[fqbn].update(create_upload_data(fqbn, installed_cores))
+        if fqbn not in skip_precomp:
```

Comment:
I would change the `skip_precom` variable name, since this is used for this too.